### DISCUSSION TITLE
Meta Thread For RPM based Linux

### DISCUSSION THREAD

**Discussion by majestrate** [oxen-io/lokinet#2098] [General]
please report any issues you encounter with rpm based linux (via rpm.oxen.io) in this thread. please include lokinet version number, os release, distro name and kernel version.

we will spin off new issues on the github issue tracker for each of them to track their progress.

**Comment by DryfterX** [comment]
https://rpm.oxen.io/ currently shows these instructions for adding the repo on Fedora:

    # Fedora:
    sudo dnf config-manager --add-repo https://rpm.oxen.io/fedora/oxen.repo

This is incorrect and should be:
`sudo dnf config-manager addrepo --from-repofile=https://rpm.oxen.io/fedora/oxen.repo`

Right now I cannot try lokinet out because I'm running Fedora Linux Asahi Remix but I wanted to add this in case anyone is trying to add the repo and having issues based on the instructions.
